src/Polytopes/Cone.jl

```diff
@@ -9,17 +15,66 @@ struct Cone #a real polymake polyhedron
 end
 function Cone(Rays::Union{Oscar.MatElem,AbstractMatrix})
     Cone(Polymake.polytope.Cone{Polymake.Rational}(
-        RAYS = matrix_for_polymake(Rays),
+        INPUT_RAYS = matrix_for_polymake(Rays),
     ))
 end
+function Cone(Rays::Union{Oscar.MatElem,AbstractMatrix}, LS::Union{Oscar.MatElem,AbstractMatrix}; non_redundant::Bool=false)
```

Comment:
Just discussed this with Claus, and we were wondering: why is there no `struct Ray` (or perhaps rather `struct Ray{T} vec::T end` ? It'd just wrap a vector, of course, but it can lead to much more intuitive code. Then this function could accept an arbitrary collection / iterator iver over `Ray` objects. Of course the "expert user" can still get constructors taking such a matrix (which here are really viewed as collections of vector representing rays, aren't they?)

Reply:
Could you maybe elaborate a little more on your thoughts here? I am afraid that we then would have to distinguish between the rays you get as a output, which are non-redundant, and the rays that you give as input. Making a constructor that eats an iterator is fine of course, but maybe just vectors would suffice?